workers/main/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@anatolyshipitz could you please extend prettier with stricter styled.
  "trailingComma": "all",
  "printWidth": 80,
  "tabWidth": 2,
  "semi": true,
  "singleQuote": true,
  "bracketSpacing": true,
  "bracketSameLine": false,
  "endOfLine": "auto",
  "quoteProps": "consistent"